interactive_ai/workflows/train/trainer/pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "otx",
     "geti-kafka-tools",
     "geti-telemetry-tools[telemetry,kafka]",
+    "setuptools==69.5.1",
```

Comment:
Based on the error message in #1296, the missing dependencies seems to be in anomalib. Have you discussed it with @samet-akcay ?

```
  File "/interactive_ai/workflows/train/trainer/.venv/lib/python3.10/site-packages/anomalib/models/video/ai_vad/clip/clip.py", line 21, in <module>
    from pkg_resources import packaging
ImportError: cannot import name 'packaging' from 'pkg_resources' (/interactive_ai/workflows/train/trainer/.venv/lib/python3.10/site-packages/pkg_resources/__init__.py)
```